capture_gui/lib.py

```diff
@@ -194,7 +195,7 @@ def _browse(path):
             default_filename = "playblast"
 
         # Default to images rule (if rule exists in workspace)
-        images_rule = mc.workspace("images", q=1, fre=True)
+        images_rule = pm.workspace.fileRules["images"]
```

Comment:
Yeah, I know. Bit of a dirty fix :worried: 

I couldn't get the cmds methods to work.